packages/solidity/contracts/StakingRewardsStore.sol

```diff
@@ -46,7 +52,7 @@ contract StakingRewardsStore is IStakingRewardsStore, AccessControl, Utils, Time
      * @param poolToken the pool token representing the rewards pool
      * @param startTime the starting time of the program
      * @param endTime the ending time of the program
-     * @param rewardRate the program's weekly rewards
+     * @param rewardRate the program's rewards rate
```

Comment:
unclear how granular it is - rate per second?

Reply:
Rate per-second, yeah. Do you want me to add this explicitly?

Reply:
in the comments? yeah

Reply:
👍